public/app/features/variables/query/QueryVariableEditor.tsx

```diff
@@ -184,63 +177,63 @@ export class QueryVariableEditorUnConnected extends PureComponent<Props, State>
   };
 
   render() {
+    const dsOptions = this.props.editor.extended?.dataSources?.length
+      ? this.props.editor.extended?.dataSources.map(ds => ({ label: ds.name, value: ds.value ?? '' }))
+      : [];
+    const dsValue = dsOptions.find(o => o.value === this.props.variable.datasource) ?? dsOptions[0];
+    const refreshOptions = [
+      { label: 'Never', value: VariableRefresh.never },
+      { label: 'On Dashboard Load', value: VariableRefresh.onDashboardLoad },
+      { label: 'On Time Range Change', value: VariableRefresh.onTimeRangeChanged },
+    ];
+    const refreshValue = refreshOptions.find(o => o.value === this.props.variable.refresh) ?? refreshOptions[0];
+    const sortOptions = [
```

Comment:
nit: move outside component since it is static?

Reply:
👍